guide/14-deep-learning/leveraging_llms_for_real_time_decision_making_in_calamities.ipynb

```diff
@@ -0,0 +1,1029 @@
+{
```

Comment:
<p>Python developers can author a custom NLP function to integrate with external models and package their model as an <strong>Esri deep learning package (.dlpk file)</strong> for use with the following tools.</p><ul><li>Use <strong>ESRI Deep Learning Package (.dlpk) file</strong> everywhere DLPK or Esri deep learning package (.dlpk file) or DLPK (Deep Learning Package) is written</li><li>Similarly Use <strong>ESRI Model Definition (.emd) file </strong>everywhere <strong>emd</strong> is written.</li></ul><p> </p><p>Spelling mistake- We will also <strong>acive </strong>the same task using our&nbsp;Python API&nbsp;interface.</p><p>Rephrase this - "We will also acive the same task using our&nbsp;<strong>Python API</strong>&nbsp;interface" to  "We will also achieve the same task using the Python API"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Fixed</p>